src/ContentRepository/Packaging/Steps/EditConnectionString.cs

```diff
@@ -10,6 +10,9 @@ public class EditConnectionString : EditXmlNodes
         public string ConnectionName { get; set; } = "SnCrMsSql";
         public string DataSource { get; set; }
         public string InitialCatalogName { get; set; }
+        public string UserName { get; set; }
+        public string Password { get; set; }
+        public bool IntegratedSecurity { get; set; } = false;

```

Comment:
Why is this property needed, why can't we calculate this from the existence of a username?